### PR TITLE
Add shortcut key for toggle tile view

### PR DESCRIPTION
**Pull request by theunafraid**

### CODE REVIEW

File: lang/main-enGB.json [hunk 1]
```diff
@@ -335,7 +335,8 @@
         "toggleFilmstrip": "Show or hide video thumbnails",
         "toggleScreensharing": "Switch between camera and screen sharing",
         "toggleShortcuts": "Show or hide keyboard shortcuts",
-        "videoMute": "Start or stop your camera"
+        "videoMute": "Start or stop your camera",
+        "toggleTileView": "Toggle tile view"
     },
```
Comment (damencho) [inline lang/main-enGB.json]: Please remove the changes from main-enGB.json, the only edited file through PRs is main.json, the rest is done through https://translate.jitsi.org.

File: react/features/toolbox/components/web/Toolbox.js [hunk 2]
```diff
@@ -475,6 +484,16 @@ class Toolbox extends Component<Props, State> {
         this.props.dispatch(toggleDialog(VideoQualityDialog));
     }
 
+    /**
+     * Dispaches an action to toggle tile view.
+     *
+     * @private
+     * @returns {void}
+     */
+    _doToggleTileView() {
```
Comment (damencho) [inline react/features/toolbox/components/web/Toolbox.js]: Is there a reason for this extra method?

Comment (theunafraid) [reply react/features/toolbox/components/web/Toolbox.js]: I followed the same pattern as for the other shortcuts, but those have analytics added. Should i remove the extra method or add analytics?

File: lang/main-enGB.json (hunk 1 above)
Comment (theunafraid) [reply lang/main-enGB.json]: Removed changes from main-enGB.json.

File: react/features/toolbox/components/web/Toolbox.js (hunk 2 above)
Comment (damencho) [reply react/features/toolbox/components/web/Toolbox.js]: I would say add analytics, so it can be tracked as the rest.

Comment (theunafraid) [reply react/features/toolbox/components/web/Toolbox.js]: Added analytics.

File: lang/main.json [hunk 3]
```diff
@@ -355,7 +355,8 @@
         "toggleScreensharing": "Switch between camera and screen sharing",
         "toggleShortcuts": "Show or hide keyboard shortcuts",
         "videoMute": "Start or stop your camera",
-        "videoQuality": "Manage call quality"
+        "videoQuality": "Manage call quality",
+        "toggleTileView": "Toggle tile view"
```
Comment (damencho) [inline lang/main.json]: Sorry, but I just noticed that we already have this string as two more keys in the translation. You can use here `toolbar.tileViewToggle`, this way there will be no need for translation of the already translated string. Thanks.

Comment (theunafraid) [reply lang/main.json]: Done, used translations of toolbar.tileViewToggle.

**Pull request merged**